spec/index.bs

```diff
@@ -15895,9 +15893,16 @@ It guarantees that:
 - if |alpha| is greater than some other |alpha1|,
     then the produced sample mask has at least as many bits set to 1 as the mask for |alpha1|
 
-### Sample frequency shading ### {#sample-frequency-shading}
+### Per-Sample shading ### {#per-sample-shading}
+<span id="sample-frequency-shading"></span>
+
+When rendering into multisampled render attachments, fragment shaders can be run once per-pixel or once per-sample.
+Fragment shaders are run once per-sample if either the `sample_index` [=builtin=] or `sample` [=interpolation sampling=]
+is used and contributes to the shader output. Otherwise fragment shaders may run once per-pixel with the result
```

Comment:
nit: emphasis
```suggestion
Fragment shaders **must** run once per-sample if either the `sample_index` [=builtin=] or `sample` [=interpolation sampling=]
is used and contributes to the shader output. Otherwise fragment shaders **may** run once per-pixel with the result
```